### PR TITLE
Condense Antidote and Antiplague item effects

### PR DESCRIPTION
**Pull request by ammalagonc**
and more lint :|

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -3091,6 +3091,12 @@
                 "UnholyWater": {
                     "Note": "Can damage only creatures with the holy trait."
                 },
+                "Version": {
+                    "Lesser": "Lesser",
+                    "Greater": "Greater",
+                    "Major": "Major",
+                    "Moderate": "Moderate"
+                },
```
Comment (pedrogrullada) [inline static/lang/re-en.json:3099]: We're not making generic labels for these adjectives for now, see this [comment thread](https://github.com/foundryvtt/pf2e/pull/16812#discussion_r1793835413)

File: build/uuid-redirects.json [hunk 2]
```diff
@@ -54,6 +54,14 @@
...
     "Compendium.pf2e.equipment-effects.Item.5oYKYXAexr0vhx84": "Compendium.pf2e.other-effects.Item.Effect: Treat Disease",
+    "Compendium.pf2e.equipment-effects.Item.QuZ5frBMJF3gi7RY": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.7z1iY4AaNEAIKuAU": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.1mKjaWC65KWPuFR4": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.mi4Md1fB2XThCand": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.R106i7WCXvHLGMTu": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.kkDbalYEavzRpYlp": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.qVKrrKpTghgMIuGH": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.HeRHBo2NaKy5IxhU": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
```
Comment (pedrogrullada) [inline build/uuid-redirects.json:64]: Try to keep the file alphabetised. We'll probably conflict, but it's easy to resolve.

File: packs/bestiary-effects/effect-aspers-elemental-aura.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Asper's Elemental Aura",
     "system": {
         "description": {
-            "value": "<p><strong>Air</strong> You gain resistance 10 to electricity damage and double your speed.</p>\n<p><strong>Fire</strong> You gain resistance 10 to fire damage. Any creature within 30 feet that receives fire damage receives an extra @Damage[2d10[persistent,fire]] damage.</p>\n<p><strong>Water</strong> You gain resistance 10 to cold damage and fast healing 20.</p>"
+            "value": "<p>You gain resistance 10 to damage of the chosen element (electricity for air, fire, or cold for water). If air, you double your speed. If fire, any creature within 30 feet of you who receives fire damage receives an extra 2d10 persistent fire damage. If water, you gain fast healing 20.</p>"
```
Comment (pedrogrullada) [inline packs/bestiary-effects/effect-aspers-elemental-aura.json:7]: This is your change from #16874, isn't it? Probably don't wanna have it here too

File: packs/bestiary-effects/effect-bosuns-command.json [hunk 4]
```diff
@@ -26,15 +26,15 @@
...
+                            "selector": "attack"
                         }
                     },
                     {
                         "label": "PF2E.Actor.Speed.Label",
                         "value": {
-                            "selector": "land-speed",
-                            "modifier": 10
+                            "modifier": 10,
+                            "selector": "land-speed"
```
Comment (pedrogrullada) [inline packs/bestiary-effects/effect-bosuns-command.json:37]: Same here

File: packs/equipment-effects/effect-antidote.json [hunk 5]
```diff
@@ -0,0 +1,125 @@
+{
+    "_id": "TjBxxlTvb6tJP1jS",
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-elixirs/antidote.webp",
+    "name": "Effect: Antidote",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Moderate)], and @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>You gain an item bonus to Fortitude saves against poisons.</p>"
```
Comment (pedrogrullada) [inline packs/equipment-effects/effect-antidote.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>You gain an item bonus to Fortitude saves against poisons.</p>"
```

File: packs/equipment-effects/effect-antiplague.json [hunk 6]
```diff
@@ -0,0 +1,125 @@
+{
+    "_id": "mVgLCbZgafIylRzh",
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-elixirs/antiplague.webp",
+    "name": "Effect: Antiplague",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Greater)] and @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Major)]</p>\n<p>You gain an item bonus to Fortitude saves against diseases.</p>"
```
Comment (pedrogrullada) [inline packs/equipment-effects/effect-antiplague.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Lesser)], @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Moderate)], @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Major)]</p>\n<p>You gain an item bonus to Fortitude saves against diseases.</p>"
```

File: packs/night-of-the-gray-death-bestiary/keznin-nevarmo.json [hunk 7]
```diff
@@ -221,7 +221,7 @@
                 "containerId": null,
                 "damage": null,
                 "description": {
-                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antiplague (Moderate)]</p>"
+                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antidote]</p>"
```
Comment (pedrogrullada) [inline packs/night-of-the-gray-death-bestiary/keznin-nevarmo.json:224]:
```suggestion
                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antiplague]</p>"
```

File: packs/npc-gallery/reckless-scientist.json [hunk 8]
```diff
@@ -645,7 +645,7 @@
                 "containerId": null,
                 "damage": null,
                 "description": {
-                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antiplague (Moderate)]</p>"
+                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antidote]</p>"
```
Comment (pedrogrullada) [inline packs/npc-gallery/reckless-scientist.json:648]:
```suggestion
                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antiplague]</p>"
```

File: packs/pfs-season-3-bestiary/reckless-akitonian-scientist.json [hunk 9]
```diff
@@ -726,7 +726,7 @@
                 "containerId": null,
                 "damage": null,
                 "description": {
-                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antiplague (Moderate)]</p>"
+                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antidote]</p>"
```
Comment (pedrogrullada) [inline packs/pfs-season-3-bestiary/reckless-akitonian-scientist.json:729]:
```suggestion
                    "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">A</span> Interact</p>\n<p>Antiplague can fortify the body's defenses against diseases. Upon drinking an antiplague, you gain a +3 item bonus to Fortitude saving throws against diseases for 24 hours; this applies to your daily save against a disease's progression.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Antiplague]</p>"
```

File: packs/bestiary-effects/effect-aspers-elemental-aura.json (hunk 3 above)
Comment (ammalagonc) [reply packs/bestiary-effects/effect-aspers-elemental-aura.json]: Good catch!

File: build/uuid-redirects.json (hunk 2 above)
Comment (ammalagonc) [reply build/uuid-redirects.json]: OK, that should be set now.

Check when you get a moment.

Thanks!

File: build/uuid-redirects.json
Comment (pedrogrullada) [inline build/uuid-redirects.json:1]: We're alphabetizing this case-insensitively!

**Pull request merged**